Resources/public/hbt/default.hbt

```diff
@@ -0,0 +1,29 @@
+<div class="content-view-line">
+    {{#each recommendations}}
+    <article class="class-article">
```

Comment:
`class-article`? I think it's very wrong naming convention for CSS classes. Please, change it.